### PR TITLE
Add test glyphs for foreground color in colored fill types

### PR DESCRIPTION
**Pull request by drott**
Make the 0xFFFF special palette index a color stop and solid fill color
for solid, linear, radial and sweep fill types. Add those as test
glyphs.

### CODE REVIEW

File: config/more_samples-glyf_colr_1.py [hunk 1]
```diff
@@ -683,6 +683,85 @@ def _composite(composite_mode, accessor_char):
...
+                ]
+            },
+            "centerX": 500,
+            "centerY": 600,
+            "startAngle": -360,
+            "endAngle": 270,
+        },
+    }
+
+    if not fill_type in fill_type_map:
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:742]: the following is more pythonic:

```suggestion
    if fill_type not in fill_type_map:
```

btw, if fill_type really is not among the expected ones, I would let the KeyError propagate. There's no point in silently returning

File: config/more_samples-glyf_colr_1.py [hunk 2]
```diff
@@ -683,6 +683,85 @@ def _composite(composite_mode, accessor_char):
     )
 
 
+def _foreground_color(fill_type, accessor_char):
+
+    fill_type_map = {
+        "solid": {
+            "Format": ot.PaintFormat.PaintSolid,
+            "PaletteIndex": 0xFFFF,
+            "Alpha": 1,
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:692]: I wonder what happens or should happen when alpha is not 1? Do you get the user-defined foreground color with some transparency applied? I see all the 0xFFFF here have alpha=1, maybe you can add one where it != 1 if this is something we support.

File: config/more_samples-glyf_colr_1.py (hunk 1 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Removed this check.

File: config/more_samples-glyf_colr_1.py (hunk 2 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Added 0.3 alpha cases, that's supported, yes. Changed the secondary color, too, for better inspection.

image

**Comment by drott** [comment]
Thanks for the review, please take another look. Feel free to go ahead and merge if it looks good to you.

File: config/more_samples-glyf_colr_1.py (hunk 2 above)
Comment (anthrotype) [reply config/more_samples-glyf_colr_1.py]: thanks!

**Pull request merged**